templates/quickstart-confluence-master-with-vpc.template.yaml

```diff
@@ -693,7 +693,7 @@ Parameters:
       It cannot start or end with a hyphen (-).
     Type: String
   QSS3KeyPrefix:
-    Default: ''
+    Default: 'quickstart-atlassian-confluence'
```

Comment:
QSS3KeyPrefix is expected to end with a `/`
See example here:
https://github.com/aws-quickstart/quickstart-atlassian-confluence/blob/develop/templates/quickstart-confluence-master-with-vpc.template.yaml#L722
https://github.com/aws-quickstart/quickstart-atlassian-confluence/blob/develop/templates/quickstart-confluence-master-with-vpc.template.yaml#L744